src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -438,7 +438,7 @@ class SharedLinkSection extends React.Component<Props, State> {
                 >
                     <PlainButton
                         {...sharedLinkSettingsButtonProps}
-                        aria-label={intl.formatMessage(messages.settingsButtonLabel)}
+                        aria-haspopup="dialog"
```

Comment:
the recommendation is to add this because it opens a new dialog but the actual behavior is determined by the `onSettingsClick` prop which is defined in the parent. so technically it could do something else like open a new page or create a dropdown menu.

however, the current intention (at least in the webapp) is to open a dialog

Reply:
would it make  sense to keep both?

Reply:
it was recommended to remove `aria-label` because it's unnecessary since the button text already acts as the "label". it also creates a mismatch between what's visible on the element (through the button text) and what the screenreader reads (through `aria-label`) because it uses different messages